python/python/lancedb/table.py

```diff
@@ -1012,6 +1012,24 @@ def drop_columns(self, columns: Iterable[str]):
             The names of the columns to drop.
         """
 
+    @abstractmethod
+    def checkout(self):
+        """Checkout a version of the table. This is an in-place operation.
```

Comment:
What does "This is an in-place operation" mean?

Why not reuse the comments from the async `checkout` / `checkout_latest` methods?